internal/gatewayapi/backendtrafficpolicy.go

```diff
@@ -377,10 +382,9 @@ func (t *Translator) translateBackendTrafficPolicyForRoute(policy *egv1a1.Backen
 
 					// some timeout setting originate from the route
 					if policy.Spec.Timeout != nil {
-						if to, err = t.buildTimeout(policy, r); err != nil {
-							return errors.Wrap(err, "Timeout")
+						if to, err = t.buildTimeout(policy, r); err == nil {
+							r.Timeout = to
```

Comment:
stop warpping error with message here to prevent the error returned by `buildTimeout` got populated again.